releasenotes/notes/bugfix-ucgate-inverse-global_phase-c9655c13c22e5cf4.yaml

```
@@ -0,0 +1,11 @@
+---
+prelude: >
+    The ``inverse()`` of the ``UCGate`` class is defined by simply inverting the
+    existing decomposition, but the inverse of the global phase was missing.
+fixes:
+  - |
+    Performs the inverse of the global phase in the ``inverse()`` function.
+  - |
+    Fixes the global phase problem of the isometry decomposition. Refer to
+    `#4687 <https://github.com/Qiskit/qiskit-terra/issues/4687>` for more
+    details.
```

The prelude is only for the final releasenote of a release, not for individual PRs 🙂 
```suggestion
fixes:
  - |
    Fix the :meth:`.UCGate.inverse` and :meth:`.Isometry.inverse` methods which 
    previously did not invert the global phase.
```

---

The problem with the isometry was not only the `inverse()` function but an error in the global phase of the decomposition. Now it is correct.

---

Ah ok, I misunderstood that! Yeah then they should be listed as two fixes (but without `prelude`)

---

Ok, thanks. I will remove the `prelude`.